### DISCUSSION TITLE
Thai TTS models

### DISCUSSION THREAD

**Discussion by K1ndred-zsq** [idiap/coqui-ai-TTS#178] [Q&A] [answered]
Thai is commonly used in Southeast Asia, have you considered adding Thai support?

**Comment by eginhard** [comment, marked as answer]
There is already a Thai model available from Fairseq that you can use with Coqui: https://github.com/idiap/coqui-ai-TTS?tab=readme-ov-file#example-text-to-speech-using-fairseq-models-in-1100-languages-

Otherwise you can also train your own model: https://coqui-tts.readthedocs.io/en/latest/tutorial_for_nervous_beginners.html If you share it, we can integrate it into Coqui.

**Reply by K1ndred-zsq** [reply]
Thank you for your response, I will try the model from Fairseq first.
